.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  # Dependencies listed in go.mod
+  - package-ecosystem: "gomod"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+
+  # Dependencies listed in .github/workflows/*.yml
+  - package-ecosystem: "github-actions"
```

Comment:
with these we'll also get the update for github action, earlier we're only getting update for go.mod. right?

Reply:
I don't remember seeing them before for either type

Reply:
yes this workflow will also update github-actions